about.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="styles/main.css">
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <header>
+    <h1>AC GILLETTE</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="projects.html">Projects</a>
+      <a href="writing.html">Writing</a>
+    </nav>
+  </header>
+  <body>
+    <section id="about">
+
```

Comment:
The hackerman photo, how did you ideally want it to display? It got stretched out in an interesting manner.